### ISSUE TITLE
obsolete Mondo terms are displayed on https://monarchinitiative.org/explore

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/monarch-app#421]
on this page, a lot of obsolete Mondo terms are displayed: https://monarchinitiative.org/explore

is there a way to exclude the obsolete terms from the display?

**Comment by cmungall** [comment]
See the guidelines on obsoletion in [10 simple rules for building an ontology-based data portal](https://docs.google.com/document/d/1906vUn0DD160zofo_CLT414_X2uqh0aoA7XmHKyIaR8/edit)

**Comment by kevinschaper** [comment]
This is unfortunately still blocked by https://github.com/Knowledge-Graph-Hub/kg-phenio/issues/90  (I think we got synonyms and xrefs, but obsolete/deprecated still isn't making it into kg-phenio)

**Comment by kevinschaper** [comment]
@caufieldjh Does this ultimately need to be fixed in kgx? It looks like `deprecated` doesn't show up in the kg-obo version of mondo either.

**Comment by kevinschaper** [comment]
Bringing the search behavior suggestion forward from Chris's doc:

> Handling obsolete terms
> Obsolete terms can be confusing. Some people recommend omitting them altogether. This is better than including them in with no visual indication, but is problematic. See comments here, which talks about obsoletes in autocomplete:
> 
> https://github.com/geneontology/amigo/issues/454 
> 
> I agree they should remain searchable. I would say there is no reason to restrict it to ID. Often a user may have in mind a term that they perceive to be in scope for GO but which we think is not, and have obsoleted. E.g histolysis. We should allow them to search for the concept and to arrive at an explanation of why we have not included as a true term.
> 
> We should have very clear behavior around obsoletes in search
> 
> obsolete results should be downweighted (ideally this would happen naturally, if we downweight less used terms)
> there should be a clear visual indication of obsoletion status. Currently we do this in the name by prefixing with "obsolete", which is good, as it guarantees we have some indication across all tools and portals, but UIs should do more, e.g strikethrough
> Regarding behavior in autocomplete. I think it's confusing to have different logic between search and AC. The user searching for an obsolete concept will expect AC results. However, given the smaller set of terms returned in AC, and the lack of context, the recommendations above are even more important, to avoid obsoletes flooding results. An obsolete should never be prioritized over a non-obsolete (perhaps except in the case of an exact match)

Which means that along with including obsolete, we need to down-weight them in the results ordering algorithm.

**Comment by caufieldjh** [comment]
I'm still seeing 3204 obsolete MONDO nodes in kg-phenio and more than 11K nodes across all namespaces - we may need to discuss this further

**Comment by caufieldjh** [comment]
@kevinschaper You're definitely right that the `deprecated` field isn't being parsed in the transform from the OWL though.

**Comment by kevinschaper** [comment]
I don't necessarily think we've done everything we can and eventually will do to handle deprecated nodes, but we have the information now, and deprecated nodes are being down-ranked so that that while they can be found, they won't bubble to the top quite so easily.  

The one piece that I wish we had addressed but didn't is explicitly noting when a node is deprecated at the top of the node page. I'll make a separate issue for that that we can take care of in the December release (or perhaps apply in an app-only patch after releasing)
